tests/Java.Interop-Tests/Java.Interop/JniRuntimeTest.cs

```diff
@@ -36,6 +36,25 @@ public void JDK_OnlySupportsOneVM ()
 				Assert.Fail ("Expected NotSupportedException; got: {0}", e);
 			}
 		}
+
+		[Test]
+		public void UseInvocationPointerOnNewThread ()
```

Comment:
This file is old, but could we start adding new tests with the pattern:
```csharp
public async Task UseInvocationPointerOnNewThread ()
{
    await Task.Run(() => ...);
}
```

Reply:
@jonathanpeppers: what's the benefit to using the `Task`-based version?  Other than ability to use `async`/`await`?

Reply:
I cringe when I see `Thread.Join()` that is the only reason, lol.